modules/ROOT/pages/dataweave-type-reuse-json-schema.adoc

```diff
@@ -0,0 +1,262 @@
+= Reusing Types from JSON Schema
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+JSON schema is a standard providing a format for what JSON data is required for a given application and how to interact with it. Many existing data formats or types are expressed using this standard. In particular, Rest APIs described by a RAML file are most likely to have their types defined in JSON schemas.
+
+In order to reuse these already defined types, you can load the JSON schema into DataWeave leaving the types structure ready to be used in your scripts.
```

Comment:
```suggestion
To reuse predefined types, you can load the JSON schema into DataWeave so that the type structure is available to be used in your scripts.
```